doc/nrf/drivers.rst

```diff
@@ -16,3 +16,4 @@ Here you can find documentation for drivers in the |NCS|, including API document
    :glob:
 
    drivers/*
+   drivers/wifi/nrf700x/*
```

Comment:
Also, the RST files except `nrf700x.rst` should be on level lower else `drivers/wifi/nrf700x/subpages` they will appear in the main menu.

Reply:
Done.